core/models/asset.py

```diff
@@ -173,6 +240,10 @@ class Asset(models.Model):
     asset_condition = models.CharField(editable=False,
                                        max_length=50,
                                        default='Brand New')
+    specs = models.ForeignKey(AssetSpecs,
+                              blank=True,
+                              null=True,
+                              on_delete=models.CASCADE)
```

Comment:
let's use `PROTECT` here..

Reply:
while we're here.. please change https://github.com/AndelaOSP/art-backend/pull/123/files#diff-bea1207c116ad66b7b398915447bd091R129 to `PROTECT` as well

Reply:
Done! Thanks for the feedback.